scripts/cli-test.sh

```diff
@@ -47,8 +47,6 @@ $FM_MINT_CLIENT fetch
 # peg out
 PEG_OUT_ADDR="$($FM_BTC_CLIENT getnewaddress)"
 $FM_MINT_CLIENT peg-out $PEG_OUT_ADDR 500
-sleep 5 # FIXME wait for tx to be included
-await_fedimint_block_sync
```

Comment:
Do we know why the `await_fedimint_block_sync` was there and why it's ok to remove? Intuitively this seems ok, I just hope there isn't some arcane dependency on it a fe lines further.

Reply:
I'm not sure why it's there. We mine more blocks a few lines lower and don't do anything with fedimint in between. So doesn't seem useful at all.